platform/mellanox/docker-syncd-mlnx-rpc/Dockerfile.j2

```diff
@@ -48,7 +37,22 @@ RUN apt-get update \
     cmake               \
     libqt5core5a        \
     libqt5network5      \
-    libboost-atomic1.71.0
+    libboost-atomic1.74.0
+
+# Build and install python-scapy
+RUN curl http://ftp.us.debian.org/debian/pool/main/s/scapy/python-scapy_2.4.0-2_all.deb --output python-scapy_2.4.0-2_all.deb \
```

Comment:
>python [](http://example.com/codeflow?start=59&length=6)

Use python3 version?

Reply:
Why is this being manually downloaded and installed here, instead of from a repo?

Reply:
This package is not available in bullseye but all its dependencies are. Until ptf can sort itself out we will need to have this workaround for RPC.

Reply:
Share the same concern with @saiarcot895.
Generally speaking, you are upgrading distro, not to add new feature or fix bugs, why introduce `python-scapy`? If it is needed, why not install by `apt`?

Reply:
Is `pip` an option?

Reply:
This package only exists on buster but we need it for PTF. 
https://github.com/p4lang/ptf/blob/36a3e3d9ac0447d28161d24028401239d756875f/debian/control#L10

I believe it was automatically getting installed as a dependency for PTF prior to this but could not be automatically installed any longer now that python-scapy is no longer in the upstream repo. 

It would be a worse hack to add buster repositories to our apt index rather than just work around and pull this manually. The problem being we need the python 2 version of this package and bullseye has removed most python2 packages. 

https://packages.debian.org/buster/python-scapy

Reply:
It looks like the following PR added python3 support for PTF. So we will be able to adjust this. However I don't wish to integrate these changes into this PR, bloating this PR with even more changes than are already in it that we would have to go and test again would be unwise. 

https://github.com/Azure/sonic-buildimage/pull/10599

Reply:
Is it possible to use [https://packages.debian.org/bullseye/python3-scapy](https://packages.debian.org/bullseye/python3-scapy)? or [https://pypi.org/project/scapy/#history](https://pypi.org/project/scapy/#history)?

Reply:
Offline discussed and this is a temporary solution.